### ISSUE TITLE
Compressed iron can't be compressed when you have the mod "Creeper Collateral" installed

### ISSUE THREAD

**Issue by BBoldt** [MineMaarten/PneumaticCraft#451]
Creeper Collateral is a mod that ensures every block drops when an explosion happens.

Problem is, it somehow prevents the in-world crafting of compressed iron from TNT.

Using:
Forge 1291
PneumaticCraft-1.7.10-1.6.7-69-universal
CreeperCollateral-[1.7.10]1.0.0B3-6
INpureCore-[1.7.10]1.0.0B9-54

**Comment by MineMaarten** [comment]
Thanks for the report, and the details on the versions you're using. I'll try to fix it.

**Comment by MineMaarten** [comment]
I've forwarded the issue to CreeperCollateral, I think it's an issue on their side.

**Comment by BBoldt** [comment]
For anyone who comes across this issue, temp fix with minetweaker:

```
// Compressed iron fix
recipes.addShaped(<PneumaticCraft:compressedIronBlock> * 2, [[null, <minecraft:iron_block>, null], [<minecraft:iron_block>, <minecraft:tnt>, <minecraft:iron_block>], [null, <minecraft:iron_block>, null]]);
```

**Comment by denoflionsx** [comment]
This should be fixed as of INpureProjects/CreeperCollateral@302e07b91bb4463168abad1fb4c476fb08ce859c. I've pushed a build to my maven for confirming the issue is resolved. The link is in the issue ticket INpureProjects/CreeperCollateral#2

Once I have confirmation that we're all good I'll push it to Curse.

**Comment by MineMaarten** [comment]
I can confirm it works now :), tnx :D
